README.md

```diff
@@ -184,7 +184,16 @@ Logging into multiple clusters via different terminal instances.
   $ export BACKPLANE_DEFAULT_OPEN_BROWSER=true
   $ ocm backplane cloud console
   `
+## Monitoring
+Monitoring command can be used to launch the specified monitoring UI.
 
+Run this command from within a cluster :
+
+```
+<prometheus/alertmanager/thanos/grafana> [flags]
+```
+
+>Note: Following version 4.11, Prometheus, AlertManager and Grafana monitoring UIs are deprecated for openshift-monitoring stack, please use 'ocm backplane console' and use the metrics tab for the same.Other monitoring stacks remain unaffected."
```

Comment:
I think it should be `Observe tab` in OpenShift console, could you double confirm?

Reply:
Please add space between `same.` and `Other`.
And also why there is an extra `"` in the end?